WhaleSpotting/react-app/src/components/Profile.tsx

```diff
@@ -0,0 +1,51 @@
+import "../styles/Profile.scss";
+import "../styles/Home.scss";
+import React from "react";
+import { useState } from "react";
+import PageNav from "./PageNav";
+
+export function Profile(): JSX.Element {
+
+    const [feedToggle, setFeedToggle] = useState("Sightings");
+
+    return (
+        <div className="body">
+            <div className="banner">
```

Comment:
I'm not sure I would call this a banner, a banner is not normally on the side. Calling it profile-pane or profile-section or something may make more sense

Reply:
Addressed.